cql3/functions/aggregate_fcts.cc

```diff
@@ -88,6 +88,10 @@ struct int128_accumulator_for {
     static T narrow(type acc) {
         T ret = static_cast<T>(acc);
         if (static_cast<type>(ret) != acc) {
+            cflogger.debug(format("SUM overflow: 0x{:08x}_{:08x}_{:08x}_{:08x} -> {}",
+                    static_cast<uint32_t>(acc >> 96), static_cast<uint32_t>(acc >> 64),
+                    static_cast<uint32_t>(acc >> 32), static_cast<uint32_t>(acc),
+                    ret).c_str());
             throw exceptions::overflow_error_exception("Sum overflow. Values should be casted to a wider type.");
```

Comment:
This format() call now happens even debug level is disabled in the log (the common case). While it should be rare, it's still not good practice to perform a heavyweight computation that is then discarded. 

It's also fine to drop this completely. If a user gets an exception, it will be easier for them to repeat the query without the sum (and perform the sum by hand) than guess which logger to enable. Logs are more for operators of the database rather than users.

Reply:
np, will remove in v3